docs/server/snippets/plugin-using-multiple.js

```diff
@@ -0,0 +1,12 @@
+//CUT
```

Comment:
This realization was one of my Formatic "ah ha" moments when it finally clicked what the lib could do.

Reply:
Yeah, it makes Formatic plugins really powerful. But it can also confuse people too. 😬